### PR TITLE
member page info

### PR DESCRIPTION
**Pull request by annamariej1015**
This is the error I am getting currently. I will keep looking but wanted you to have the code as well
TypeError: Cannot read property 'role' of undefined

We need the second div for the images of the whole team below the single person card.

### CODE REVIEW

File: src/pages/Member.js [hunk 1]
```diff
@@ -1,9 +1,118 @@
-const Member =() =>{
+import { useEffect, useState } from 'react';
+import { membersData } from '../data/teamMembers';
+import { useParams } from 'react-router-dom';
+import { FaStar, FaRegStar } from 'react-icons/fa';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {faGithub, faLinkedin} from '@fortawesome/free-brands-svg-icons';
+import { faEnvelope } from '@fortawesome/free-solid-svg-icons';
+
+const languages = [
```
Comment (meru20) [inline src/pages/Member.js]: I don't think we need languages here since we can render it from the array

File: src/pages/Member.js [hunk 2]
```diff
@@ -1,9 +1,118 @@
...
+                            </div>
+                        
+                            <div className='card-body'>
+                                <h4 className='card-title'>{member.firstName} {member.lastName}</h4>
+                                <p className='card-text'>Bio: {member.bio}</p>
+                            
+                                    <div className='my-2'>
+                                        <strong>Languages:</strong>
+                                        <p className='ml-5'>
+                                            {languages.map((language,i) => {
```
Comment (meru20) [inline src/pages/Member.js]: we may need to do {member.language.map } instead of {language.map} to render and match our language data from the array.

**Review by meru20 — APPROVED** [review]
Looks Good!

**Review by annamariej1015 — COMMENTED** [review]
Looks good!

**Pull request merged**